prometheus-to-sd/translator/prometheus.go

```diff
@@ -64,6 +65,27 @@ func getPrometheusMetrics(config *config.SourceConfig) (*PrometheusResponse, err
 	return &PrometheusResponse{rawResponse: string(body)}, nil
 }
 
+func doPrometheusRequest(url string, auth config.AuthConfig) (resp *http.Response, err error) {
+	// Allow insecure https connections
+	client := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+	}
+	request, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return nil, err
+	}
+	if len(auth.Username) > 0 {
+		request.SetBasicAuth(auth.Username, auth.Password)
+	} else if len(auth.Token) > 0 {
+		request.Header.Set("Authorization", "Bearer "+auth.Token)
```

Comment:
please use fmt.Sprintf() instead of string concatenation

Reply:
Fixed.